shared/chat/conversation/messages/wrapper/container.js

```diff
@@ -38,6 +39,17 @@ const mapStateToProps = (state: TypedState, {messageKey, prevMessageKey}: OwnPro
   const isFirstNewMessage = !!(conversationState && message && message.messageID && conversationState.get('firstNewMessageID') === message.messageID)
   const prevMessage = getMessage(state, prevMessageKey)
   const skipMsgHeader = prevMessage && prevMessage.type === 'Text' && prevMessage.author === author
+
+  const firstMessageEver = !prevMessage
```

Comment:
we show it if its the actual first message, or the top message while we're loading, or if there's enough time between us and our neighbor